udata_front/theme/gouvfr/templates/reuse/list.html

```diff
@@ -39,40 +40,66 @@
-                <a
-                    href="{{ url_for('reuses.list') }}"
-                    class="fr-tag{% if not reuses.query._filters.topic %} active{% endif %}">
-                    {{ _('All') }}
-                </a>
-            </li>
-            {% for topic_id, label in reuses_topics %}
-                <li>
-                    <a
-                        href="{{ reuses.query.to_url(url, replace=True, **{'topic': topic_id}) }}"
-                        class="fr-tag{% if reuses.query._filters and reuses.query._filters.topic and reuses.query._filters.topic.topic == topic_id %} active{% endif %}"
-                    >
-                        {{ label }}
-                    </a>
-                </li>
-            {% endfor %}
-        </ul>
+    <div class="fr-pt-3v">
+        <form method="GET" class="fr-grid-row">
+            <div class="fr-search-bar fr-search-bar--lg w-100" role="search">
+                <label class="fr-label" for="search-reuses">
+                    {{ _('Search...') }}
+                </label>
+                <input
+                    name="q"
+                    class="fr-input"
+                    placeholder="{{ _('Search...') }}"
+                    type="search"
+                    id="search-reuses"
+                    value="{{reuses.query._query}}"
+                />
+                <button class="fr-btn" title="{{_('Search')}}" type="submit">
+                    {{ _('Search') }}
+                </button>
+            </div>
+            <div class="fr-grid-row fr-py-5v fr-displayed-lg">
+                <ul class="fr-tags-group">
+                    <li>
+                        <a
+                            href="{{ url_for('reuses.list') }}"
+                            class="fr-tag{% if not reuses.query._filters.topic %} active{% endif %}">
+                            {{ _('All') }}
+                        </a>
+                    </li>
+                    {% for topic_id, label in reuses_topics %}
+                        <li>
+                            <a
+                                href="{{ reuses.query.to_url(url, replace=True, **{'topic': topic_id}) }}"
+                                class="fr-tag{% if reuses.query._filters and reuses.query._filters.topic and reuses.query._filters.topic.topic == topic_id %} active{% endif %}"
+                            >
+                                {{ label }}
+                            </a>
+                        </li>
+                    {% endfor %}
+                </ul>
+            </div>
+            <div class="w-100 fr-grid-row fr-mt-7v fr-pt-3v justify-between align-items-center fr-grid-row--middle border-top border-default-grey">
+                <p class="fr-m-0">{{ _('%(result)s results', result=reuses.total) }}</p>
+                {{sort_search(reuses, sorts)}}
+            </div>
+        </form>
     </div>
     <ul class="search-results fr-grid-row fr-grid-row--gutters mt-lg-md">
```

Comment:
We need a bit of spacing on search results if no result is found on reuse search page.
![image](https://user-images.githubusercontent.com/28541902/164232173-97861f6c-13fd-4b5f-a5f3-080626974489.png)